modules/farm_fd2/src/entrypoints/cover_crop_seeding/lib.submit.unit.cy.js

```diff
@@ -0,0 +1,392 @@
+    cy.restoreSessionStorage();
+  });
+
+  afterEach(() => {
+    cy.saveLocalStorage();
+    cy.saveSessionStorage();
+  });
+
+  it('Check the asset--plant', () => {
+    expect(results.plantAsset.type).to.equal('asset--plant');
+    expect(results.plantAsset.attributes.name).to.equal(
+      form.date + '_' + form.crops.join('_')
+    );
+    expect(results.plantAsset.attributes.notes.value).to.equal(form.comment);
+    expect(results.plantAsset.relationships.plant_type).to.have.length(2);
+    expect(results.plantAsset.relationships.plant_type[0].id).to.equal(
+      cropMap.get(form.crops[0]).id
+    );
+    expect(results.plantAsset.relationships.plant_type[1].id).to.equal(
+      cropMap.get(form.crops[1]).id
+    );
+    expect(results.plantAsset.relationships.parent).to.have.length(0);
+  });
+
+  it('Check the area seeded quantity--standard', () => {
+    expect(results.areaSeededQuantity.type).to.equal('quantity--standard');
+    expect(results.areaSeededQuantity.attributes.measure).to.equal('ratio');
+    expect(results.areaSeededQuantity.attributes.value.decimal).to.equal(
+      form.areaSeeded
+    );
+    expect(results.areaSeededQuantity.attributes.label).to.equal('Area Seeded');
+    expect(results.areaSeededQuantity.relationships.units.id).to.equal(
+      unitMap.get('PERCENT').id
+    );
+    expect(results.areaSeededQuantity.relationships.units.type).to.equal(
+      'taxonomy_term--unit'
+    );
+    expect(results.areaSeededQuantity.attributes.inventory_adjustment).to.be
+      .null;
+    expect(results.areaSeededQuantity.relationships.inventory_asset).to.be.null;
+  });
+
+  it('Check the log--seeding', () => {
+    expect(results.seedingLog.type).to.equal('log--seeding');
+    expect(results.seedingLog.attributes.name).to.equal(
+      form.date + '_cs_' + form.crops.join('_')
+    );
+    expect(results.seedingLog.attributes.timestamp).to.contain(form.date);
+
+    expect(results.seedingLog.relationships.location.length).to.equal(3);
+    expect(results.seedingLog.relationships.location[0].id).to.equal(
+      fieldMap.get(form.location).id
+    );
+    expect(results.seedingLog.relationships.location[1].id).to.equal(
+      bedMap.get(form.beds[0]).id
+    );
+    expect(results.seedingLog.relationships.location[2].id).to.equal(
+      bedMap.get(form.beds[1]).id
+    );
+
+    expect(results.seedingLog.relationships.category.length).to.equal(2);
+    expect(results.seedingLog.relationships.category[0].id).to.equal(
+      categoryMap.get('seeding').id
+    );
+    expect(results.seedingLog.relationships.category[1].id).to.equal(
+      categoryMap.get('seeding_cover_crop').id
+    );
+
+    expect(results.seedingLog.relationships.asset[0].id).to.equal(
+      results.plantAsset.id
+    );
+
+    expect(results.seedingLog.relationships.quantity.length).to.equal(1);
+    expect(results.seedingLog.relationships.quantity[0].id).to.equal(
+      results.areaSeededQuantity.id
+    );
+  });
+
+  if (form.winterKill) {
```

Comment:
What is the purpose of the `if` here? Seems like you are submitting a static form where `form.winterKill` will be `true`.   Can `form.winterKill` be `false`?  If not then remove the `if`.  If so, then can it be made more obvious why this `if` is here?